gtsfm/two_view_estimator.py

```diff
@@ -60,6 +60,7 @@ def __init__(
         bundle_adjust_2view: bool,
         eval_threshold_px: float,
         bundle_adjust_2view_maxiters: int = 100,
+        ba_reproj_error_thresh: float = 0.5,
```

Comment:
Isn't this too strict?

Reply:
For reference, we use 3px/4px threshold in verification and the final BA.

Reply:
I'll let @johnwlambert comment, as he told me he fine-tuned these for his thesis. But this is 2-view bundle adjustment, which also optimizes for calibration, I think, so it should be able to overfit very closely. In reichstag it does not filter a lot of correspondences away.